rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -45,18 +45,22 @@ class RecordFixture : public TemporaryDirectoryFixture
 public:
   RecordFixture()
   {
-    root_bag_path_ = (rcpputils::fs::path(temporary_dir_path_) / "bag").string();
-    storage_path_ = (rcpputils::fs::path(root_bag_path_) / "bag_0").string();
-    database_path_ = storage_path_ + ".db3";
-    std::cout << "Database " << database_path_ << " in " << temporary_dir_path_ << std::endl;
+    set_path("bag");
   }
 
-  void SetUp() override
+  void set_path(const std::string & bagname)
```

Comment:
Any reason for the method syntax change? https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
`SetUp()` is a method specific to [GTest](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests) that is usually overridden to setup a fixture. `set_path()` is a method we introduced.